docs/snippets/vue/button-story-default-docs-code.vue3.js.mdx

```diff
@@ -0,0 +1,41 @@
+```js
+// Button.stories.js
+
+import Button from './Button.vue';
+
+export default {
+  title: 'Button',
+  component: Button,
+  //👇 Creates specific argTypes
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
+};
+
+//👇 Some function to demonstrate the behavior
+const someFunction = (someValue) => {
+  return `i am a ${someValue}`;
+};
+
+export const ExampleStory = (args) => {
+  //👇 Assigns the function result to a variable
+  const functionResult = someFunction(args.label);
+  return {
+    components: { Button },
+    setup() {
+      //👇 The args will now be passed down to the component
+      return {
+        ...args,
+        //👇 Replaces arg variable with the override (without the need of mutation)
+        label: functionResult,
```

Comment:
```suggestion
      return {
        args: {
          ...args,
          //👇 Replaces arg variable with the override (without the need of mutation)
          label: functionResult,
        }
```

`args` needs to be a key inside the object returned from `setup()` if you want to `v-bind="args"` in the template.

Reply:
@phated fixed with the latest commit. Thanks for the clarification.